modules/samples/connectedcup/feature/connectedcup-feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.type-view/public/js/download.js

```diff
@@ -146,7 +146,7 @@ function downloadAgent() {
     payload.owner = $inputs[1].value;
```

Comment:
Use double quoted object key access for reading JSON attributes as it's considered to be more reliable.
Ex: person.name as person["name"]